src/main/java/org/jabref/logic/importer/fetcher/CrossRef.java

```diff
@@ -137,6 +137,10 @@ private BibEntry jsonItemToBibEntry(JSONObject item) throws ParseException {
                             .map(year -> Integer.toString(year)).orElse("")
             );
             entry.setField(StandardField.DOI, item.getString("DOI"));
+            entry.setField(StandardField.JOURNAL, item.getString("container-title"));
+            entry.setField(StandardField.PUBLISHER, item.getString("publisher"));
+            entry.setField(StandardField.ISSUE, item.getString("issue"));
```

Comment:
Caused by: kong.unirest.json.JSONException: JSONObject["issue"] not found.
  Needs to be optString